File: src/types/case.ts
export interface DiseaseCase {
  id: string;
  disease: string;
  city: string;
  cases: number;
  timestamp: string;
  lat: number;
  lng: number;
}

export interface DiseaseInputData {
  disease: string;
  city: string;
  cases: string | number;
  timestamp: string;
}

File: backend/models/case.py
from pydantic import BaseModel
from datetime import datetime

class DiseaseCaseBase(BaseModel):
    disease: str
    city: str
    cases: int
    timestamp: str

class DiseaseCaseCreate(DiseaseCaseBase):
    pass

class DiseaseCase(DiseaseCaseBase):
    id: str
    lat: float
    lng: float

    class Config:
        from_attributes = True